lib/bike_club.rb

```diff
@@ -1,3 +1,23 @@
 class BikeClub
+  attr_reader :name, :bikers
+  
+  def initialize(name)
+    @name = name
+    @bikers = []
+  end
 
+  def add_bikers(biker)
+    @bikers << biker
+  end
+
+  def most_rides
+    hash = Hash.new(0)
+    @bikers.each do |biker|
+      hash[biker] = biker.rides.values.flatten.count
+    end
+    value = hash.max_by do |biker_object, ride_count|
+      ride_count
+    end
+    value[0].name
```

Comment:
Great work! Here's another option
```
    @bikers.max_by do |biker|
      biker.rides.sum do |ride, times|
        times.count
      end
    end
```

Reply:
But doing this in 2 steps is totally okay! I would just name `value` more descriptively -- maybe `biker` since it ends up being a Biker object.